pyproject.toml

```diff
@@ -11,7 +11,7 @@
 [build-system]
 requires = [
     "setuptools_scm[toml]>=6.2",
-    "scikit-build-core[pyproject]",
+    "scikit-build-core[pyproject]==0.5.1",
```

Comment:
please leave this out... It builds fine with later versions, just not editable installs :)

Reply:
Yes, I will remove it :+1: